lib/react/jsx/babel_transformer.rb

```diff
@@ -0,0 +1,29 @@
+require 'babel/transpiler'
+module React
+  module JSX
+    class BabelTransformer
+      DEPRECATED_OPTIONS = [:harmony, :strip_types, :asset_path]
+
+      def initialize(options)
+        _options = options.dup
+        has_old_opts = DEPRECATED_OPTIONS.map do |option|
```

Comment:
Is there any downside to passing the deprecated keys to Babel? Are they ignored or do they raise errors?

If they're just ignored, this could be simplified a bit:

```ruby
if (options.keys & DEPRECATED_OPTIONS).any?
  # deprecation warning ...
end
```